README.md

```diff
@@ -47,6 +47,7 @@ Here are some commands you can run:
 | yarn amo:dev                | Start the dev server/proxy (for amo) using data from the dev server (https://addons-dev.allizom.org/) |
 | yarn amo:no-proxy           | Start the dev server without a proxy (for amo) using data from Docker |
 | yarn amo:stage              | Start the dev server/proxy (for amo) using data from the staging server (https://addons.allizom.org/) |
+| yarn check                  | Run [nsp][] to check dependencies |
```

Comment:
This should have a more descriptive name, so `nsp-check` or something. `check` is a bit too generic.